<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_dados</class>
 <widget class="QDialog" name="fm_dados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1021</width>
     <height>491</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Venda&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Arquivo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(22, 41, 85);
color :#000

</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1">
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>931</width>
       <height>229</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Source Code Pro</family>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">
color: #009164;</string>
              </property>
              <property name="text">
               <string> Produto   </string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_produto">
              <property name="font">
               <font>
                <family>Source Code Pro</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="accessibleDescription">
               <string/>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #009164;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Source Code Pro</family>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #009164;</string>
              </property>
              <property name="text">
               <string>  Valor    </string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_preco">
              <property name="font">
               <font>
                <family>Source Code Pro</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #009164;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Source Code Pro</family>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #009164;</string>
              </property>
              <property name="text">
               <string>Quantidade</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_quantidade">
              <property name="font">
               <font>
                <family>Source Code Pro</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #009164;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_adicionar">
          <property name="font">
           <font>
            <family>Source Code Pro</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #009164;</string>
          </property>
          <property name="text">
           <string>Adicionar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="font">
           <font>
            <family>Source Code Pro</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #009164;</string>
          </property>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Produto</string>
           </property>
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="background">
            <color>
             <red>38</red>
             <green>76</green>
             <blue>116</blue>
            </color>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Pre√ßo</string>
           </property>
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantidade</string>
           </property>
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_produtoSort">
            <property name="font">
             <font>
              <family>Source Code Pro</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #009164;</string>
            </property>
            <property name="text">
             <string>Ordenar por Produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_precoSort">
            <property name="font">
             <font>
              <family>Source Code Pro</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #009164;</string>
            </property>
            <property name="text">
             <string>Ordenar por Pre√ßo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_quantidadeSort">
            <property name="font">
             <font>
              <family>Source Code Pro</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #009164;</string>
            </property>
            <property name="text">
             <string>Ordenar por Quantidade</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>931</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <family>Source Code Pro</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ForbiddenCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #009164;</string>
        </property>
        <property name="title">
         <string>Valor total da compra</string>
        </property>
        <widget class="QLabel" name="txt_total">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>30</y>
           <width>211</width>
           <height>81</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="font">
         <font>
          <family>Source Code Pro</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ForbiddenCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #009164;</string>
        </property>
        <property name="title">
         <string>Menor pre√ßo</string>
        </property>
        <widget class="QLabel" name="txt_menorValor">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>30</y>
           <width>211</width>
           <height>81</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Source Code Pro</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <family>Source Code Pro</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #009164;</string>
        </property>
        <property name="title">
         <string>Maior pre√ßo</string>
        </property>
        <widget class="QLabel" name="txt_maiorValor">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>30</y>
           <width>211</width>
           <height>81</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Source Code Pro</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2">
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>140</y>
       <width>371</width>
       <height>201</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #009164;</string>
     </property>
     <property name="title">
      <string>Dados</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="btn_salvar">
        <property name="font">
         <font>
          <family>Source Code Pro</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #009164;</string>
        </property>
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_carregar">
        <property name="font">
         <font>
          <family>Source Code Pro</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #009164;</string>
        </property>
        <property name="text">
         <string>Carregar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
